setup.py

```diff
@@ -54,6 +54,7 @@
             "descartes>=1.0.0",
             "geopandas>=0.8.0",
             "shapely>=1.6.0",
+            "pyarrow>=14.0.0",
```

Comment:
why this dependency?

Reply:
It is required to by `pandas` to load parquet files. I had it installed locally already when I installed pandas. But the CI did not have it installed and complained that it could not read parquet files without it.